tests/test_basic.py

```diff
@@ -44,6 +57,24 @@ def test_failed_jobs(self):
         response = self.client.get('/failed')
         self.assertEqual(response.status_code, HTTP_OK)
 
+    def test_worker_python_version_field(self):
+        w = Worker(['q'])
+        w.register_birth()
+        response = self.client.get('/workers.json')
+        data = json.loads(response.data.decode('utf8'))
+        if getattr(w, 'python_version', None):
+            self.assertEqual(w.python_version, data['workers'][0]['python_version'])
+        pop_connection()
```

Comment:
Since It's pushed in setUp, then It will be better to pop It in tearDown.

Reply:
Yeah, it makes sense. Edited.